### DISCUSSION TITLE
集群版本的milvus配置etcd为1个pod

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#21614] [Q&A and General discussion]
![image](https://user-images.githubusercontent.com/117245915/211475465-2991cb81-05b2-4ecb-aed9-d861e42e6d56.png)

使用milvus-helm的master分支部署的milvus集群，values.yaml默认是集群版的etcd，默认3台pod。
现在想配置etcd为单pod，不需要高可用，把etcd的replicaCount设置成1后，etcd服务起不来。难道

![image](https://user-images.githubusercontent.com/117245915/211476229-e23739de-c355-4587-b6b3-eecdb46c2d1d.png)

etcd日志：
![image](https://user-images.githubusercontent.com/117245915/211476040-4bfe29c9-3c6d-4fc6-803d-164c6a460a43.png)

**Comment by LoveEachDay** [comment]
@RongHouchao Your etcd cluster has been provisioned to with 3 members. You cannot simply change the replica count to 1 for etcd statefulset. Could you run `kubectl describe pods my-release-etcd-0` to show the pod manifests?

**Reply by RongHouchao** [reply]
kubectl describe pods my-release-etcd-0:

![image](https://user-images.githubusercontent.com/117245915/211794366-62066373-c391-4868-9a25-8e8b92a3fe98.png)

那如果就是不需要使用高可用的etcd，就只要一台pod就可以，这个需要怎么配置呢

**Reply by LoveEachDay** [reply]
Here the env: `ETCD_INITIAL_CLUSTER` has three members, you'd only keep the `my-release-etcd-0`.
